indico/modules/events/management/controllers.py

```diff
@@ -178,3 +178,13 @@ def _update_session_coordinator_privs(self, form):
             log_msg = 'Session coordinator privilege changed to {}: {}'.format(form.data[priv_field],
                                                                                COORDINATOR_PRIV_TITLES[priv_field])
             self.event_new.log(EventLogRealm.management, EventLogKind.positive, 'Protection', log_msg, session.user)
+
+
+class RHMoveEvent(RHConferenceModifBase):
```

Comment:
And it's clearly an event action - IMHO it'd be pretty weird to have this in the categories module. Not to mention the potential import hell since now you need to import the event management base RH inside the category module